lib/presentation/auth/widgets/enter_username.dart

```diff
@@ -1,139 +1,240 @@
+          builder: (context, state) {
+            return Column(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                Row(
+                  children: const [
+                    Expanded(
+                      child: Text(
+                        'How should we call\r\n you?',
+                        style: TextStyle(
+                          fontWeight: FontWeight.w700,
+                          fontSize: 32.0,
+                        ),
+                        maxLines: 2,
+                        textAlign: TextAlign.center,
+                      ),
+                    )
+                  ],
                 ),
-              ],
-            ),
-            const SizedBox(height: 45.0),
-            Form(
-              key: _formKey,
-              onChanged: () => setState(
-                () => _isNameValid = _formKey.currentState?.validate() == true,
-              ),
-              child: TextFormField(
-                controller: _usernameController,
-                onChanged: (username) => _username = username,
-                style: const TextStyle(fontSize: 20.0),
-                keyboardType: TextInputType.text,
-                decoration: InputDecoration(
-                  hintText: 'Preferred name',
-                  helperText: "Use your real name or choose a user name",
-                  focusColor: kAccentColor,
-                  enabledBorder: OutlineInputBorder(
-                    borderRadius: BorderRadius.circular(20.0),
-                    borderSide:
-                        const BorderSide(width: 0, color: Colors.transparent),
-                  ),
-                  focusedBorder: OutlineInputBorder(
-                    borderRadius: BorderRadius.circular(20.0),
-                    borderSide:
-                        const BorderSide(width: 0, color: Colors.transparent),
+                const SizedBox(height: 10.0),
+                Row(
+                  children: const [
+                    Expanded(
+                      child: Text(
+                        'Enter your first name or use a recognizable name that others can identify you by',
+                        textAlign: TextAlign.center,
+                        style: TextStyle(color: kInactiveColor),
+                      ),
+                    ),
+                  ],
+                ),
+                const SizedBox(height: 45.0),
+                Form(
+                  key: _formKey,
+                  onChanged: () => setState(
+                    () => _isNameValid =
+                        _formKey.currentState?.validate() == true,
                   ),
-                  focusedErrorBorder: OutlineInputBorder(
-                    borderRadius: BorderRadius.circular(20.0),
-                    borderSide:
-                        const BorderSide(width: 0, color: Colors.transparent),
+                  child: Column(
```

Comment:
Could add a TODO to cleanup, since some of the code looks duplicated.

Reply:
Yeah good idea, will probably just fix it though.